### PR TITLE
zh-translation:/docs/setup/upgrade/cni-helm-upgrade/index.md

### PR DESCRIPTION
**Pull request by donychen1134**
ref:https://github.com/servicemesher/istio-official-translation/issues/1383

### CODE REVIEW

**Comment by donychen1134** [comment]
@googlebot I signed it!

**Comment by donychen1134** [comment]
@googlebot I fixed it.

**Comment by donychen1134** [comment]
@googlebot I fixed it.

File: content/zh/docs/setup/upgrade/cni-helm-upgrade/index.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Istio **不支持** 跨版本升级。仅支持从 {{< istio_previous_version >}
 
 ## 升级步骤 {#Upgrade steps}
 
-[下载新版本 Istio](/docs/setup/#downloading-the-release)，并切换目录到新版本的目录下。
+[下载新版本 Istio](/docs/setup/getting-started/#download)，并切换目录到新版本的目录下。
```
Comment (rootsongjc) [inline content/zh/docs/setup/upgrade/cni-helm-upgrade/index.md]: 所有链接前加 `/zh` 前缀。

Comment (donychen1134) [reply content/zh/docs/setup/upgrade/cni-helm-upgrade/index.md]: fixed.

**Comment by tanjunchen** [comment]
@donychen1134  @vivian99-wu @rootsongjc  I think squash commits looks fine，16 commits  look a little bit.

**Pull request merged**